### DISCUSSION TITLE
aggressive issue closing

### DISCUSSION THREAD

**Discussion by Codinak** [joe-bell/cva#203] [Feedback]
Are you guys up to hold the 0 open issues so strong, that you:

1. don't properly read issue description
2. don't want any reply and lock down issues closed in the matter of seconds?

https://github.com/joe-bell/cva/issues/202

You got even the fix for your wrong typing provided, that excludes the claim of using both `class` and `className` at the same time, because the fix tests that this cases properly ERROR.

The fix basically has all the test cases for you written.

The example does not use both. The problem was not understood and silenced.

**Comment by Codinak** [comment]
The answer proves that the issue was not read entirely, as not a single thing in the reply is true when facing the posted issue and fix proposal. How can you 1. close and 2. silence the issue like that? I won't ever waste any second anymore searching for issue cause and propose fixes and explanation for them, if they are going to be flushed down the drain in a matter of 10 seconds.

**Comment by joe-bell** [comment]
I've been working on this project solidly for almost 2 years without pay; in the last 6 months alone I have been tirelessly pushing `cva@1.0`

On top of that, managing issues, and responding to feedback, I have a full-time job and a personal life

@akomm I apologise for prematurely closing your issue and appreciate you taking the time to fix

I'm only human, and I'm trying my best

**Reply by Codinak** [reply]
And how did closing the issue ignoring its content helped you? By this logic, just close up the library? Did I nag that you are not fixing it in 10 seconds? I did not. I've just spent time:

1. exclude its not an error on my side
2. figure out why the issue happen
3. figure out a solution
4. explain why it is an issue and why it happens, post all the test cases commented

Because you maintain a library my time and effort can be flushed down the toilet now like that? I don't see the logic. It just signals people to not invest time into the library (which will help you how?)

**Reply by joe-bell** [reply]
@Codinak I have apologised to @akomm, I am exploring the fix, and I offered you a truthful answer

**Reply by akomm** [reply]
Just to clarify, it was a mistake on my side:

Codinak is my purely private account I've "freshly" created to separate from the one I've mixed privat and work. 
I just forgot to switch back to akomm.

Just realized it on a second glance.

**Reply by joe-bell** [reply]
I guess it's a day of learning from our mutual mistakes 😂

**Reply by joe-bell** [reply]
Shipped, will be available under `cva@beta` – more to follow on that shortly
